docs/EVE-LOGGING.md

```diff
@@ -35,3 +40,19 @@ When a direct ssh connection to a device is possible, /persist/rsyslog/syslog.tx
 ## Log export to cloud
 
 Rsyslogd is configured to send logs via TCP socket to logmanager. Logmanager bundles the logs into protobuf messages and exports them to cloud using API. Today when a device loses network connectivity to cloud, there is a back-off mechanism built into the logmanager that prompts rsyslogd to stop sending logs. When there is network drop, logmanager stops reading log messages from rsyslogd (TCP socket). Rsyslogd stops sending logs when it's output socket buffer is full. As a result following logs will be queued on disk. After the device's network connectivity is restored, rsyslogd starts sending logs from it's disk queues.
+
+## Log files still present in device
+
+1. Logmanager's log file present in /persist/log/logmanager.log
+   Latest code with cron, logrotate removed from pillar this log file grows irregulated.
+   What can be done to control the total size of this file on disk?
```

Comment:
Shouldn't this be an issue in github as opposed to in the document? We clearly need to address it soon.

Reply:
Actually let me file a tracker story at least and assign it to @gkodali-zededa

Reply:
I'll start on this story.